src/elements/content-sidebar/activity-feed/task-new/__tests__/TaskDueDate.test.js

```diff
@@ -13,6 +13,6 @@ describe('elements/content-sidebar/ActivityFeed/task-new/TaskDueDate', () => {
     test('should add proper class when past due date', () => {
         const dueDate = shallow(<TaskDueDate dueDate={new Date() - 1000} status="NOT_STARTED" />);
 
-        expect(dueDate.find('.bcs-is-taskOverdue')).toHaveLength(1);
+        expect(dueDate.find('.bcs-TaskDueDate')).toHaveLength(1);
```

Comment:
This is testing for a different thing now. Before we were checking for overdue date, and now it's looking for the proper wrapper (and not the state of the new pill).